File: domain/src/main/java/com/studentschedulleapp/androidclient/domain/models/UserTokens.java
package com.studentschedulleapp.androidclient.domain.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserTokens {
    private Long userId;
    private String accessToken;
    private String refreshToken;
}

File: domain/src/main/java/com/studentschedulleapp/androidclient/domain/models/tokens/UserTokens.kt
package com.studentschedulleapp.androidclient.domain.models.tokens;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserTokens {
    private Token accessToken;
    private Token refreshToken;
}
